### DISCUSSION TITLE
Couldn't open device problem (Windows, v3.3.3)

### DISCUSSION THREAD

**Discussion by domeniqo** [OpenHantek/OpenHantek6022#386] [General]
Hi, I have a brand new Hantek 6022BE, but since the very beginning, I have been unable to get any signal from this device. A driver seems to be installed correctly, but the device does not work either with the original SW or with OpenHantek. In OpenHatek SW, after selecting the device from the drop-down list, error note appears for ~0.5 sek and nothing else happens. According to error text, it look like status from libusb_open is not LIBUSB_SUCCESS and the upload of firmware is not successful.

tag 3.3.3, file/line **openhantek\src\usb\uploadFirmware.cpp:31**

Do you think it could be a faulty device or is there anything else that comes to your mind that could solve this issue?

Thank you for any help and advice!

**Comment by nelsonflorez-hub** [comment]
Usen USBDview y eliminé toda referencia a un drivers para ese osciloscopio y reinstale de nuevo.

No use la versión 3.3.3 genera una pantalla negra, use la versión 3.4.0

**Reply by Ho-Ro** [reply]
> Usen USBDview y eliminé toda ...

**English please!** I translated it for you:

> Use USBDview and delete all references to drivers for that oscilloscope, then reinstall.
Do not use version 3.3.3 as it causes a black screen; use version 3.4.0 instead.
